integreat_cms/cms/views/push_notifications/push_notification_form_view.py

```diff
@@ -69,32 +70,57 @@ def get(self, request, *args, **kwargs):
             kwargs.get("language_slug"), only_active=True
         )
 
-        push_notification = PushNotification.objects.filter(
+        push_notification = region.push_notifications.filter(
             id=kwargs.get("push_notification_id")
         ).first()
 
-        if push_notification and push_notification.sent_date:
-            messages.info(
-                request,
-                __(
-                    _(
-                        "This news message has already been sent as push notification to mobile devices."
+        other_regions, all_languages, disable_edit = map(
+            extract_pn_details(request, push_notification).get,
+            ("other_regions", "all_languages", "disable_edit"),
+        )
```

Comment:
I might be damaged from destructuring objects in JavaScript. I didn't go for a tuple  because I wanted to get values by name, not by their position.
This was a last minute change only because pylint complained that the function was two lines too long